frontend/src/screens/ContactScreen.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react' 
+
+const ContactScreen = () => {
+
+    return (
+        <main className="row center">
+            <div className="contact-container">
+                <div className="center-mid">
+                    <h1 className="style-text">Hello!</h1>
+                    <h1 className="style-text">Welcome to Vinyl Cakes</h1>
+                    <p>____________________________</p>
```

Comment:
There is a semantic tag that would probably be more suitable for you - `<hr />`, check it out!